test/unit/org/apache/cassandra/index/sai/iterators/AbstractKeyRangeIteratorTest.java

```diff
@@ -15,17 +15,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.cassandra.index.sai.utils;
+package org.apache.cassandra.index.sai.iterators;
 
 import java.util.Arrays;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import org.junit.Assert;
 
+import org.apache.cassandra.index.sai.utils.PrimaryKey;
+import org.apache.cassandra.index.sai.utils.SaiRandomizedTest;
 import org.apache.cassandra.utils.Pair;
 
-public class AbstractRangeIteratorTest extends SaiRandomizedTest
+public class AbstractKeyRangeIteratorTest extends SaiRandomizedTest
```

Comment:
We might want to make this class `abstract` and perhaps rename it to `AbstractKeyRangeIteratorTester`.

Reply:
It is not `abstract` in ASF. Thus I am hesitant to do this change in this PR.

Reply:
At the same time the name is `Tester` in ASF. Also `SaiRandomizedTest` is a Tester. So I will look to this.

Reply:
The inheritance tree is different in ASF than here. The root in ASF is `CQLTester`, while the root in CC is `RandomizedTest`, which is from a third-party library. Thus I am hesitant to do this change and prefer to postpone.

Reply:
Fine with me.